pincer/commands/__init__.py

```diff
@@ -1,7 +1,10 @@
 # Copyright Pincer 2021-Present
 # Full MIT License can be found in `LICENSE` at the project root.
 
-from .commands import command, user_command, message_command, ChatCommandHandler
+from .commands import (
+    command, user_command, message_command, ChatCommandHandler, hash_app_command,
```

Comment:
Why not just make these all 1 per line as its already started wrapping?

Reply:
I ran black on it so its fixed now :)